ember/app/ui/components/identity-addresses/component.js

```diff
@@ -21,24 +22,40 @@ export default class IdentityAddressesComponent extends Component {
   @service notification;
   @service store;
   @service intl;
-  // @service can;
 
   // Lifecycle
 
-  @action onUpdate() {
+  @action
+  onUpdate() {
+    this.fetchCountries.perform();
     this.fetchAddresses.perform(this.args.identity);
   }
 
   // List
 
   @lastValue("fetchAddresses") addresses;
-
-  @restartableTask *fetchAddresses(identity) {
+  @restartableTask
+  *fetchAddresses(identity) {
     return yield this.store.query("address", {
       filter: { identity: identity.id },
     });
   }
 
+  @lastValue("fetchCountries") countries;
+  @restartableTask
+  *fetchCountries() {
```

Comment:
Why the need for a custom fetch?

Reply:
We fetch the countries list from the backend instead for consistency with django-countries instead of using a js package or similar